services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/health.py

```diff
@@ -14,11 +18,17 @@
     },
 )
 async def health_endpoint(
-    application_health: ApplicationHealth = Depends(get_application_health),
+    application_health: Annotated[ApplicationHealth, Depends(get_application_health)],
```

Comment:
what is `application_health` then for? are you using it anywhere else (cannot find it except for the tests)? I would have expected that `application_health` groups all checks of the dynamic-sidecar app.

Reply:
it is used inside the user services containers are created. If these are not started the health is set to failed.